blueprints/ember-sky-labels/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  description: 'Copy SkyLabels SCSS into the host application',
+
+  afterInstall: function() {
+    var import = '@import "sky-labels";';
```

Comment:
Expected an identifier and instead saw 'import' (a reserved word).<br>'import' is defined but never used.